pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -223,9 +233,9 @@ func (c *Cluster) removeExtraMgrs(daemonIDs []string) {
 	}
 }
 
-// ReconcileMultipleServices reconciles the services if the active mgr is the one running
+// ReconcileActiveMgrServices reconciles the services if the active mgr is the one running
 // in the sidecar
-func (c *Cluster) ReconcileMultipleServices(daemonNameToUpdate string, mgrStatSupported bool) error {
+func (c *Cluster) ReconcileActiveMgrServices(daemonNameToUpdate string, mgrStatSupported bool) error {
```

Comment:
Since `ReconcileActiveMgrServices` is a method we don't need to pass `c.clusterInfo.CephVersion.IsAtLeastPacific()` right? We can just call it.

Reply:
When it's called from the sidecar we don't have access to the full ceph version. To clarify this restriction, I should change it so this method is not on the `(c *Cluster)` type.

Reply:
And we cannot populate it?

Reply:
Yeah populating it is simpler anyway, will do that